src/components/NavigationAccountExpandCollapse.vue

```diff
@@ -48,5 +48,11 @@ export default {
 			}
 		},
 	},
+	mounted() {
+		let self = this
+		this.$el.ondragenter = function() {
```

Comment:
pro tip: if you use an arrow function `() => {...}` `this` won't be bound to the element but the surrounding one from the Vue component